.github/workflows/_build_test_upload.yml

```diff
@@ -270,11 +270,18 @@ jobs:
           conda activate conda_build_env
           conda install -yq conda-build -c conda-forge
           packaging/build_conda.sh
+          conda index ./conda-bld
       - name: Install TorchData Conda Package
         shell: bash -l {0}
-        run: conda install --offline conda-bld/*/torchdata-*.tar.bz2
+        run: |
+          if [[ ${{ needs.get_release_type.outputs.type }} == 'official' ]]; then
+            CONDA_CHANNEL=pytorch
+          else
+            CONDA_CHANNEL=pytorch-${{ needs.get_release_type.outputs.type }}
+          fi
+          conda install pytorch torchdata cpuonly -c $(pwd)/conda-bld/ -c "$CONDA_CHANNEL"
```

Comment:
In order to install `torchdata` with a local conda package, I need to specify the `CONDA_CHANNEL` the dependencies like `pytorch`.